templates/accounts/describe_sounds.html

```diff
@@ -125,6 +125,29 @@
             $("#id_" + idx0 + "-zoom").val(Math.floor(zoom));
         }
 
+        var centerMapTimers = {};
+        var centerMapTimeout = 1000; // In milliseconds
+
+        function centerMapToLatLonZoomFields(n) {
+            // To avoid too many updates of the map, only re-sync it with the contents of the lat, lon and zoom fields
+            // after centerMapTimeout has passed since the last change event
+
+            var idx = parseInt(n, 10);
+
+            if (centerMapTimers[idx] !== undefined){
+                clearTimeout(centerMapTimers[idx]);
+            }
+
+            centerMapTimers[idx] = setTimeout(function (){
+                var idx0 = idx - 1; // Get map number
+                var lat = $("#id_" + idx0 + "-lon").val();
+                var lon = $("#id_" + idx0 + "-lat").val();
+                var zoom = $("#id_" + idx0 + "-zoom").val();
+                maps[idx].setZoom(parseInt(zoom, 10) - 1);
+                maps[idx].setCenter([parseFloat(lat), parseFloat(lon)]);
```

Comment:
if I delete all of the text from a field, I get `lng_lat.js:29 Uncaught Error: Invalid LngLat object: (NaN, 23.8858)`

Reply:
I can enter the letter e. It looks like it's so that exponent notation like 1e3 works, but e alone gives an error (because it's not text)

Reply:
I can enter numbers greater than 180, and it seems to accept it and move the map, but I no longer see the red marker appear

Reply:
Yeah I saw these errors, for some reason browser is not allowing numbers only. I could in fact enter other characters. Maybe it depends on the browser. However there is no problem if one of these values can't be parsed correctly as the map won't update. When the form is later validated in server (or by browser when clicking submit) then the errors will be highlighted.

Reply:
the zoom input has up/down arrows on it, but doesn't allow me to go below 11, but I can type a number in and it's fine. I don't know what the acceptable range is here.
If I enter a negative zoom number I get a warning on the console `Geometry exceeds allowed extent, reduce your vector tile buffer size` and it resets to 1.

Reply:
>  can enter numbers greater than 180, and it seems to accept it and move the map, but I no longer see the red marker appear

ok, this is an error. maybe doing some basic validation in the `var x = $("#id_" + idx0 + "-x").val();` would help.

> the zoom input has up/down arrows on it, but doesn't allow me to go below 11, but I can type a number in and it's fine. I don't know what the acceptable range is here.
If I enter a negative zoom number I get a warning on the console Geometry exceeds allowed extent, reduce your vector tile buffer size and it resets to 1.

one thing is the range the map accepts and the other the one Freesound accepts (we ask for a minimum zoom). It is normal that the map updates because you also want to see small zooms (while you're searching for some place for example). again, when form is validated later user wil be warned if zoom value is not correct.

Reply:
If I move the map with the mouse I get a huge number of decimal places, e.g. `19.464546208718573`. This should be reduced

Reply:
when I click/drag the map I can't change the latitude bigger than 36 or lower than -36

Reply:
if I zoom with the mouse wheel, the map moves around a bit, depending on where the cursor is, causing the lat/long values to change. if it's possible to make the mapbox map stay still when zooming then that'd be great

Reply:
it'd be nice to have instead of the "remove geotag" checkbox, a link which hides the map again

Reply:
Most of these comments addressed in e3ecf82f2509410535259b316c52ca19dc8e7a6f